src/metabase/pulse/render/body.clj

```diff
@@ -348,12 +348,27 @@
         default-x-type (if (isa? (:effective_type x-col) :type/Temporal)
                          "timeseries"
                          "ordinal")]
-    {:colors (public-settings/application-colors)
-     :x      {:type (or (:graph.x_axis.scale _viz-settings) default-x-type)
-              :format x-format}
-     :y      {:type (or (:graph.y_axis.scale _viz-settings) "linear")
-              :format y-format }
-     :labels labels}))
+    {:colors   (public-settings/application-colors)
+     :stacking (if (:stackable.stack_type viz-settings) "stack" "none")
+     :x        {:type (or (:graph.x_axis.scale viz-settings) default-x-type)
+                :format x-format}
+     :y        {:type (or (:graph.y_axis.scale viz-settings) "linear")
+                :format y-format }
+     :labels   labels}))
+
+(defn- set-default-stacked
+  "Default stack type is stacked for area chart with more than one metric.
+   So, if :stackable.stack_type is not specified, it's stacked.
+   However, if key is explicitly set in :stackable.stack_type and is nil, that indicates not stacked."
+  [viz-settings card]
+  (let [is-stacked     (if (contains? viz-settings :stackable.stack_type)
```

Comment:
`is-stacked` reads so strangely in a language that allows questions marks in identifiers. Minor nit but `stacked?` is far preferable

Reply:
it feels like `stacked?` is an actual predicate function, tho, which this one is not. I will change it, tho

Reply:
Also, this logic here is careful to check that if there is a value at `(:stackable.stack_type viz-settings)` that it needs to be equal to `"stacked"`. But this subtlety is dropped for `default-y-pos` which only cares if there is _any_ truthy value at that location.

Reply:
> it feels like `stacked?` is an actual predicate function, tho, which this one is not. I will change it, tho

That's fair. I think some people make a big distinction about functions which return booleans vs boolean values themselves. I don't make a huge different. I think its a minor quibble so you can leave it alone if you like. But the `is-` prefix just stands out to me a lot. No big deal though.

Reply:
possible values are `stacked`, `normalized` and nil. normalized is not implemented in FE yet and will require a fair bit of work in BE also.

Reply:
actually, christ

`grep -nr '!= ' . | wc`
     331    2416   29835